workspaces/src/rpc/client.rs

```diff
@@ -527,7 +525,7 @@ where
     Retry::spawn(retry_strategy, task).await
 }
 
-pub(crate) async fn send_tx(
+pub async fn send_tx(
```

Comment:
Making these methods public will require more time to review and make sure that the interfaces are aligned and documented.